website/cypress/support/commands.ts

```diff
@@ -47,6 +47,11 @@ Cypress.Commands.add("signInUsingEmailedLink", (emailAddress) => {
     const loginLink = emails.pop().html.match(/href="[^"]+(\/api\/auth\/callback\/[^"]+?)"/)[1];
     cy.visit(loginLink);
     cy.url().should("include", "/dashboard");
+
+    // we do a GET to this url to force the python backend to add an entry for our user
+    // in the database, otherwise the tos acceptance will error with 404 user not found
+    // then we accept the tos
+    cy.request("GET", "/api/available_tasks").then(() => cy.request("POST", "/api/tos", {}));
```

Comment:
I could not find any cleaner option, I know we need to refactor our user / migrate our users db to the python backend eventually. the sooner the better.

Reply:
Yeah I need to get onto updating the backend routes to handle the JWT.  Once we do that we can use the web api route for just auth stuff for the short term.